packages/cadl-vscode/package.json

```diff
@@ -121,9 +121,9 @@
     "@types/mocha": "~10.0.0",
     "@types/node": "~18.11.9",
     "@types/vscode": "~1.53.0",
-    "@cadl-lang/compiler": "~0.37.0",
-    "@cadl-lang/eslint-config-cadl": "~0.4.1",
-    "@cadl-lang/internal-build-utils": "~0.3.1",
+    "@cadl-lang/compiler": "workspace:~0.37.0",
```

Comment:
Can you check if prepare-publish gets broken by these.

Reply:
Made the cadl-azure PR and updated the prepare-publish to work with this https://github.com/Azure/cadl-azure/pull/2319